run_build.sh

```diff
@@ -1,11 +1,33 @@
 #!/bin/bash
 
+# Extra arguments to pass to DAOS scons
 EXTRA_BUILD="${1}"
 
 export BUILD_DIR="<path_build_area>" #e.g./scratch/POC/BUILDS/
 export MPICH_DIR="<path_to_mpich>" #e.g./scratch/POC/mpich
 export OPENMPI_DIR="<path_to_openmpi>" #e.g./scratch/POC/openmpi
 
+# DAOS branch to clone
+DAOS_BRANCH="master"
+
+# Optional DAOS commit to checkout after clone
+DAOS_COMMIT=""
+
+# Optional DAOS branches to merge
+DAOS_BRANCHES=()
+
+# Optional DAOS commits to cherry-pick
+DAOS_CHERRY=()
+
+# DAOS scons build type
+DAOS_BUILD_TYPE="release"
+
+# MPI to build DAOS with
+MPI_TARGET="mvapich2"
+
+# Optional IOR commit to checkout after clone
+IOR_COMMIT=""
+
```

Comment:
Ideally, this will allow us to configure a build without needing to "manually" alter the build commands below